MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/IMvxRecyclerAdapterBindableHolder.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using MvvmCross.Droid.Support.V7.RecyclerView.Model;
+
+namespace MvvmCross.Droid.Support.V7.RecyclerView
+{
+    public interface IMvxRecyclerAdapterBindableHolder
+    {
+        event Action<MvxViewHolderBindedEventArgs> MvxViewHolderBinded;
```

Comment:
`Binded` is weird.  I think it needs to be something with `Bound` in it (and drop the `Mvx` bit) like `ViewHolderBound` or something.